cmd/autoheal/alerts_worker.go

```diff
@@ -219,9 +219,10 @@ func (h *Healer) runRule(rule *autoheal.HealingRule, alert *alertmanager.Alert)
 	// Execute the action:
 	switch typed := action.(type) {
 	case *autoheal.AWXJobAction:
```

Comment:
can we have the rule have a "runner" field, so we can just do `rule.run()` ?
I think it will be much more fun then this case ...

Reply:
Sure, good suggestion!

Reply:
I'd prefer to avoid that and keep the rule object as a pure container of data, without logic.

Reply:
I tend to agree with @jhernand